core/src/main/scala/org/apache/spark/deploy/FaultToleranceTest.scala

```diff
@@ -113,7 +112,7 @@ private object FaultToleranceTest extends App with Logging {
     assertValidClusterState()
 
     killLeader()
-    delay(30 seconds)
+    delay(30.seconds)
```

Comment:
I think it might be reasonable to get rid of so that we can turn on the warnings to catch useful-ish things (like the "x isDefined")